Source/WebKit/UIProcess/API/glib/WebKitUserContent.cpp

```diff
@@ -433,3 +437,122 @@ API::ContentRuleList& webkitUserContentFilterGetContentRuleList(WebKitUserConten
     ASSERT(userContentFilter);
     return *userContentFilter->contentRuleList;
 }
+
+/**
+ * WebKitUserScriptMessage: (ref-func webkit_user_script_message_ref) (unref-func webkit_user_script_message_unref)
+ *
+ * A helper class that sends user provided script to the signal handler.
+ *
+ * Since: 2.38
```

Comment:
We are a bit late in the 2.38 development cycle (we have branched a few weeks ago) so maybe we need to leave this new feature for 2.40&mdash;although I would love to have it in 2.38 and given it's a new addition it will definitely not break existing applications and the risk of cherry-picking this to the 2.38 branch should be very close to none.

@carlosgcampos WDYT, should we try to have this ready soon for 2.38, or do we leave for 2.40?